src/atc/schema_manager/schema_manager.py

```diff
@@ -107,7 +110,3 @@ def get_all_spark_sql_schemas(self) -> Dict[str, str]:
             str_schemas[name] = str_schema
 
         return str_schemas
-
-
```

Comment:
@Okami1 I removed this function because it was used to execute code at the time of importing. Executing code in this way leads to slow software start time and is bad practice. Please see the function initSchemaManager that I implemented in the test section. Such a function should exist in your project to populate the SchemaManger.